api/v1beta1/application_types.go

```diff
@@ -27,6 +27,14 @@ const (
 	ReasonInit = "Init"
 )
 
+// Constants for status
```

Comment:
Why do we need this change?

Reply:
Because KALM wants to reference to the constants. The application golang library only exposes the `api` package, but not the `pkg` package.